examples/quickstart.rs

```diff
@@ -19,23 +19,26 @@ mod errors {
 
 use errors::*;
 
-fn main() {
-    if let Err(ref e) = run() {
-        println!("error: {}", e);
-
-        for e in e.iter().skip(1) {
-            println!("caused by: {}", e);
-        }
-
-        // The backtrace is not always generated. Try to run this example
-        // with `RUST_BACKTRACE=1`.
-        if let Some(backtrace) = e.backtrace() {
-            println!("backtrace: {:?}", backtrace);
-        }
-
-        ::std::process::exit(1);
-    }
-}
+fn main() {                                                                         
+    if let Err(ref e) = run() {                                                     
+        use ::std::io::Write;                                                       
+        let (stderr, errmsg) = (&mut ::std::io::stderr(), "Error writing to stderr");
```

Comment:
Use two `let` declarations, pattern matching doesn't add any value in this case.
Also, I'm not sure you need the `&mut` part.